src/globals.css

```diff
@@ -167,6 +167,8 @@
     --orange-700: hsl(var(--color-orange-700));
     --orange-800: hsl(var(--color-orange-800));
     --orange-900: hsl(var(--color-orange-900));
+    --shadow-sidebar: 0px 33px 32px -16px rgba(0, 0, 0, 0.1),
```

Comment:
we should start using variables for the shadows once they are included in the style guide. I have created this one for now and will add the others as they become available